index.html

```diff
@@ -4567,7 +4567,16 @@
     <section id="rtcrtpcodeccapability*">
       <h3>dictionary RTCRtpCodecCapability</h3>
       <p>The <dfn><code>RTCRtpCodecCapability</code></dfn> dictionary provides
-      information on the capabilities of a codec.</p>
+        information on the capabilities of a codec. Exactly one <code>RTCRtpCodecCapability</code>
+        will be present for each supported combination of parameters that must be identical
+        between the offer and answer. For example:</p>
+      <ol>
+        <li>Exactly one "opus" codec, with <code>numChannels</code> set to 2,
```

Comment:
@taylor-b Yes, that is how things work in ORTC.  Is there a situation where it would make sense to have 2 channels with stereo=0?